src/uct/ib/base/ib_device.h

```diff
@@ -349,9 +349,9 @@ int uct_ib_device_is_port_roce(uct_ib_device_t *dev, uint8_t port_num);
 
 
 /**
- * @return 1 if the gid_raw is 0, 0 otherwise.
+ * @return 1 if the gid is valid, 0 otherwise
```

Comment:
(consistency w above funcs) can simplify the doc to "return whether.."

Reply:
changed it